src/main/java/net/dv8tion/jda/internal/JDAImpl.java

```diff
@@ -675,6 +676,53 @@ public SnowflakeCacheView<RichCustomEmoji> getEmojiCache()
         return CacheView.allSnowflakes(() -> guildCache.stream().map(Guild::getEmojiCache));
     }
 
+    @Nonnull
+    @Override
+    public RestAction<ApplicationEmoji> createApplicationEmoji(@Nonnull String name, @Nonnull Icon icon)
+    {
+        Checks.inRange(name, 2, 32, "Emoji name");
```

Comment:
The `32` should be a constant, you can reference the constant in javadocs using `{@value ApplicationEmoji#EMOJI_NAME_MAX_LENGTH}`. This improves the maintainability and makes docs consistent.